docs/dask.md

```diff
@@ -1,54 +1,81 @@
+"mintpy.networkInversion.numWorkers = all" will allocate `multiprocessing.cpu_count()` number of workers to the dask computation.
 
-**walltime** - this is the maximum amount of your time your job is permitted to run on a compute node before it is killed by the node. Walltimes are required so as to ensure users and jobs cannot hog nodes for extended periods of time without doing any substantial computations. Most queues have maximum allowed compute times, after which still running jobs will be terminated, but this is often on the order of 48+ hours. It is recommended to use relatively short walltimes for `MintPy` (the test dataset uses `wall time: 00:30:00` (30 minutes)), but not too short, or your job may never connect to the dask scheduler and finish its computation (when in doubt, start small and work towards larger wall times as needed). Note that the walltime format changes slightly depending on the job scheduler that is in place on your HPC system. For `LSF` schedulers, walltime is accepted and expected to be in `HH:MM` format, while `SLURM` and `PBS` schedulers only accept walltime in `HH:MM:SS` format.  
+### Test on Stampede2 ###
 
-**queue** - this the scheduler queue to submit your jobs to. Most systems have several different queues with different amounts and types of resources you can request. For this reason, it is important to check with your HPC cluster admin or documentation to determine the most appropriate queue to `MintPy` on and what resources are available on that queue. Requesting for more resources (core, memory, nodes, etc.) on a queue than are available often leads to your jobs being rejected from queue, and can, on some systems, lead to fines or account suspensions.
+To show the run time improvement, we test three datasets (Galapagos, Fernandina, and Kilauea) with different number of cores on a compute node in the [Stampede2 cluster's skx-normal queue](https://portal.tacc.utexas.edu/user-guides/stampede2#overview-skxcomputenodes). Results are as below:
+ 
+| Property              | Fernandina Dataset | Galapagos Dataset | Kilauea Dataset            |
+|-----------------------|--------------------|-------------------|----------------------------|
+| Input File Size       | 0.623 GB           | 0.233 GB          | 15.0 GB                    |
+| Dask Configuration    | None               | None              | None                       |
+| Cluster Memory Used   | 16 GB              | 16 GB             | 16 GB (64 GB for 1-2 cores)|
+| Chunk Size Used       | 100e6 (1 GB)       | 100e6 (1 GB)      | 100e6 (1 GB)               | 
+ 
```

Comment:
Could you also add the info of matrix shape, i.e. 500 by 2000 by 1700?

Reply:
1. Could you add the matrix shape info for the other two datasets?

2. I don't get what is the "cluster memory used"? Is it the memory you assigned to the node? I don't think any of the 3 datasets requires this much memory.

Reply:
Hi @Ovec8hkin, what you added is the dimension of the output timeseries.h5 file, could you add the dimension of the input ifgramStack.h5 file? `info.py` would print out that info.

Reply:
Whoops, thought those were the same thing. Prior to block-by-block local cluster required a certain amount of memory be allocated to each dask worker. I picked something relatively large (16GB) to test with, since that approximately the size of the Kilauea dataset. This does not appear to be needed anymore with block-by-block implemented.

Reply:
What parameters from info.py specify the dimensions?

Reply:
no option needed.